grpc-proto/proto/query.proto

```diff
@@ -506,6 +506,10 @@ message CasWriteUnknown {
 message Row {
   // The values for the columns (in the same order as ResultSet.columns).
   repeated Value values = 1;
+  // The paging state for the row (only set on enriched responses).
```

Comment:
So this is a change to the main gRPC protos to enable a bridge feature? I'm not sure we want to do that. An alternative to isolate this new feature to bridge protos/impl only would be to update the bridge `ExecuteEnrichedQuery` to return an `EnrichedResponse` containing `EnrichedRows`?

Reply:
I'm also a bit surprised the `bridge.proto` is in the `grpc-proto` module, but I know that was introduced prior to this PR.

Reply:
If I make this return EnrichedRows/EnrichedResultSet, and use EnrichedQuery (which was my initial approach), is there a way for codegen to have EnrichedRow essentially extend Row? When I was initially doing things that way I noticed it requires a lot of code duplication in a bunch of Handlers all the way down; I could attempt it that way to maintain separation. Though I do think that this belongs in grpc "core" since it is really just using C*'s RowDecorator API

Reply:
I agree with Jeff and see the pain of Eric wrt to extending stuff.. However, I am up for not changing the public API, this should not be part of what the end users should use nor receive!

Reply:
Can we have a row wrapper? That wraps the existing Row and add the two byte properties?

Also, new messages should not be part of the `query.prot`o that's being public messages imo, we should add some king of `bridge-query.proto` or something like this..